air-script/src/cli/transpile.rs

```diff
@@ -1,39 +1,29 @@
+use clap::Args;
 use std::{fs, path::PathBuf};
-use structopt::StructOpt;
 
 use codegen_winter::CodeGenerator;
 use ir::AirIR;
 use parser::parse;
 
-#[derive(StructOpt, Debug)]
-#[structopt(
-    name = "Transpile",
-    about = "Transpile AirScript source code to Rust targeting Winterfell"
-)]
-pub struct TranspileCmd {
-    /// Path to input file
-    #[structopt(short = "i", long = "input", parse(from_os_str))]
-    input_file: Option<PathBuf>,
-    /// Path to output file
-    #[structopt(short = "o", long = "output", parse(from_os_str))]
-    output_file: Option<PathBuf>,
+#[derive(Args)]
+pub struct Transpile {
+    input: PathBuf,
+
+    #[arg(
+        short,
+        long,
+        help = "Output filename, defaults the input file with the .rs extension"
+    )]
+    output: Option<PathBuf>,
```

Comment:
minor nit: the "help" on line 15 should probably say "... default to the ..." instead of "defaults the".

Also, it seems like we've lost some info here - i.e., `structop` would take doc comments and convert them into command hints (i.e., for the input file it would say "Path to input file" etc.). Could we not do something similar with `clap`?

Reply:
Updated it, now it looks like this:

```
AirScript language compiler

Usage: airc <COMMAND>

Commands:
  transpile
  help       Print this message or the help of the given subcommand(s)

Options:
  -h, --help     Print help
  -V, --version  Print version
```

Reply:
Could we also add the description of the `transpile` command? I think it was "Transpile AirScript source code to Rust targeting Winterfell" previously.

Reply:
changed:

```
AirScript language compiler

Usage: airc <COMMAND>

Commands:
  transpile  Transpile AirScript source code to Rust targeting Winterfell
  help       Print this message or the help of the given subcommand(s)

Options:
  -h, --help     Print help
  -V, --version  Print version
```